socorro/external/es/supersearch.py

```diff
@@ -214,6 +222,8 @@ def get(self, **kwargs):
                         results_number = param.value[0]
                     elif param.name == '_facets_size':
                         facets_size = param.value[0]
+                    elif param.name == '_histogram_interval.date':
+                        histogram_interval_date = param.value[0]
```

Comment:
Yep, that's because all values are turned into lists, even when they have and can have only one value. I'll add a comment.